VRDR/DeathRecord.cs

```diff
@@ -7285,7 +7290,7 @@ public string InjuryDescription
                 {
                     CreateInjuryIncidentObs();
                 }
-                InjuryIncidentObs.Value = new FhirString(value);
+                InjuryIncidentObs.Value = new CodeableConcept(null, null, null, value);
```

Comment:
Does this not need a system/code value (or display text)?

Reply:
Nope.  The key idea is that we are passing a string value in the value.text field, because it is entirely possible that somebody will want to use the coded content of the value.   For now, all NCHS wants is a text string.
So, the new CodeableConcept(null, null, null, value); is creating  value.text only, and leaving value.coding.code empty.